pyonfx/shape.py

```diff
@@ -525,8 +526,12 @@ def split(self, max_len: float = 16, tolerance: float = 1.0) -> Shape:
         **Tips:** *You can call this before using :func:`map` to work with more outline points for smoother deforming.*
 
         Parameters:
-            max_len (int or float): The max length that you want all the lines to be
-            tolerance (float): Angle in degree to define a bezier curve as flat (increasing it will boost performance during reproduction, but lower accuracy)
+            max_len (int or float):
+                The max length that you want all the lines to be
+
```

Comment:
Blank line?

Reply:
I added some blank lines but I'm not sure if the doc build won't complain about it... Do you want me to revers the docstring changes?